keylime/tenant.py

```diff
@@ -871,13 +918,19 @@ def do_reglist(self) -> Optional[Dict[str, Any]]:
         if not self.registrar_ip or not self.registrar_port:
             raise UserError("registrar_ip and registrar_port have both to be set in the configuration")
 
+        self.set_full_id_str()
+
         response = registrar_client.doRegistrarList(
             self.registrar_ip, self.registrar_port, tls_context=self.tls_context
         )
 
+        # Marked for deletion (need to modify the code on CI tests)
         logger.info(
-            "From registrar %s port %s retrieved %s", self.registrar_ip, self.registrar_port, json.dumps(response)
+            "From registrar %s port %s retrieved %s\n", self.registrar_ip, self.registrar_port, json.dumps(response)
```

Comment:
should the \n be removed?

Reply:
The plan is to leave it there during this "transition" between the old and new messages